code4romania/teacher-workout-ios
# Lesson finished screen

### What does it fix?


Closes #57  



### How has it been tested?





1. Start a lesson
2. Tap the continue button
3. Check we display the Finished lesson screen

![Simulator Screen Shot - iPhone 13 mini - 2021-10-30 at 12 30 03](https://user-images.githubusercontent.com/481425/139527700-3d160082-e386-40e8-a6cb-32d81280e02d.png)
